go/worker/common/committee/runtime_host.go

```diff
@@ -164,21 +163,7 @@ func (n *computeRuntimeHostNotifier) watchPolicyUpdates() {
 				SignedPolicyRaw: raw,
 			}}
 
-			var response *protocol.Body
-			call := func() error {
-				resp, err := n.host.Call(n.ctx, req)
-				if err != nil {
-					n.logger.Error("failed to dispatch RPC call to runtime",
-						"err", err,
-					)
-					return err
-				}
-				response = resp
-				return nil
-			}
-
-			retry := backoff.WithMaxRetries(backoff.NewConstantBackOff(retryInterval), maxRetries)
-			err := backoff.Retry(call, backoff.WithContext(retry, n.ctx))
+			response, err := n.host.Call(n.ctx, req)
```

Comment:
can also remove the `retryInterval` & `maxRetries` vars

Reply:
:+1: